pkg/datarepair/checker/checker.go

```diff
@@ -19,3 +30,107 @@ func (c *Config) Run(ctx context.Context) (err error) {
 
 	return err
 }
+
+
+// Checker contains the information needed to do checks for missing pieces
+type Checker struct {
+	params *pb.IdentifyRequest
+	pointerdb storage.KeyValueStore
```

Comment:
I'm not sure we want direct access to the underlying database of pointerdb. This will cause hidden coupling in terms of how data is stored in the database. The requests should go against the pointerdb API.

Reply:
Although that might need to happen eventually, I would prefer we do it against the API for now.